packages/docs/docs/self-hosting/install-on-aws.md

```diff
@@ -31,69 +66,96 @@ You will need permission to access the following AWS services:
 
 You will also need to setup your [AWS credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
-## Create an SES email address
+### Setup SES
 
 It is **strongly** recommended to setup an SES email address with production access, meaning that it can send email to any email recipient. Email is used to verify identities, send login instructions, and handle password reset requests.
 
 Follow the [Creating and verifying identities in Amazon SES](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html) guide to register an email address for system generated emails.
 
-## Deploy Bot Lambda Layer
+## Setup a config repo
 
-**Optional:** If you intend to use Medplum Bots, you will need an [AWS Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-concepts.html#gettingstarted-concepts-layer).
+### Create your config repo
 
-You can use the `deploy-bot-layer.sh` script to automate this process:
+Medplum recommends creating a separate git repository and npm project to manage your CDK infra-as-code. This repository will only contain JSON configuration files.
 
 ```bash
-./scripts/deploy-bot-layer.sh
+mkdir my-medplum-cdk-config
+cd my-medplum-cdk-config
+npm init -y
 ```
 
-See the [Creating and sharing Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html) guide to learn more.
+Medplum recommends pushing this git repository to your source control provider such as GitHub or GitLab.
 
-## Run the `init` tool
+### Add CDK dependencies
 
-Most AWS resources are automatically created using CDK, but some either cannot or are not recommended. Use the Medplum `init` tool to setup those resources and build the config file.
+If you have not already done so, add the common AWS CDK dependencies. This includes all of the base CDK capabilities and constructs.
 
 ```bash
-cd packages/cdk
-npm run init
+npm i aws-cdk-lib cdk constructs
+```
+
+### Add Medplum CDK dependency
+
+Add the Medplum CDK dependency. This includes the Medplum CDK construct.
+
+```bash
+npm i @medplum/cdk
```

Comment:
Hi, I do not think this have been published properly as I get the following -> 

![image](https://user-images.githubusercontent.com/1785595/224990704-9f37f56e-eb9c-4d75-a681-728530b36bb8.png)